### PR TITLE
added routes and changed project structure

### PR DESCRIPTION
**Pull request by VKong6019**
- added routing to the http requests for users and letters
- moved a file or two into a new directory

### CODE REVIEW

**Comment by nicoledanuwidjaja** [comment]
🚢🚢🚢🚢🚢🚢

Jk fix it loser

File: services/futureLetterManager.js [hunk 1]
```diff
@@ -84,32 +82,32 @@ function addLetterToUser(json, field) {
...
+};
 
 
 
 
 // __________________LETTER METHODS__________________
 // create a letter
-function createLetter(json) {
+exports.createLetter = function(json) {
     letters.add({
```
Comment (nicoledanuwidjaja) [inline services/futureLetterManager.js:150]: no

File: services/futureLetterManager.js [hunk 2]
```diff
@@ -135,37 +133,20 @@ function getLetter(json) {
...
-function getReplyLetter(json) {
+exports.getReplyLetter = function(json) {
     return letters.doc(json['id']).get().get('replyLetters');
-}
-
-
-
-
-// // Your web app's Firebase configuration
-// var firebaseConfig = {
```
Comment (nicoledanuwidjaja) [inline services/futureLetterManager.js:160]: hide our secrets here please

Comment (nicoledanuwidjaja) [reply services/futureLetterManager.js:160]: delete your commits and start over

File: index.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+const functions = require('firebase-functions');
```
Comment (nicoledanuwidjaja) [inline index.js]: yes this one isn't right

File: index.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+const express = require('express');
+const app = express();
+const futureLetterService = require('./services/futureLetterService.js');
+const bodyParser = require("body-parser");
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+// get requests
```
Comment (nicoledanuwidjaja) [inline index.js:15]: i think you should try to test this with front end haha you're too sexy no

File: services/futureLetterManager.js (hunk 2 above)
Comment (VKong6019) [reply services/futureLetterManager.js:160]: we're not using these anyways

File: services/futureLetterManager.js (hunk 1 above)
Comment (VKong6019) [reply services/futureLetterManager.js:150]: there's no problem here

File: index.js (hunk 4 above)
Comment (VKong6019) [reply index.js:15]: can't even read a pr

**Pull request merged**